python/ray/autoscaler/commands.py

```diff
@@ -257,6 +257,16 @@ def get_or_create_head_node(config, config_file, no_restart, restart_only, yes,
             init_commands = config["head_setup_commands"]
             ray_start_commands = config["head_start_ray_commands"]
 
+        ray_start_cmd = list(
+            filter(lambda x: "ray start" in x, ray_start_commands))
+        if len(ray_start_cmd) and not \
+                any("autoscaling-config" in x for x in ray_start_cmd):
+            logger.warning(
+                "Ray start on the head node does not have the flag"
+                "--autoscaling-config set. The head node will not launch"
+                "workers. Add --autoscaling-config=~/ray_bootstrap_config.yaml"
+                "to ray start in the head_start_ray_commands section.")
```

Comment:
Can we pull this into a helper function? e.g., `warn_about_bad_start_command`

Reply:
Actually, it would also be useful to warn if "ray start" is missing..

Reply:
Will do!